files/helm/fluent-operator/common.yaml

```diff
@@ -1,6 +1,13 @@
 containerRuntime: containerd
 operator:
   priorityClassName: system-cluster-critical
+  resources:
+    limits:
+      cpu: 100m
+      memory: 60Mi
```

Comment:
remove limits, it mostly causes issues. Its better to monitor usage and adapt accordingly